src/renderer/components/watch-video-chapters/watch-video-chapters.css

```diff
@@ -35,11 +34,9 @@
 
 .chapter {
   display: grid;
-  grid-template-areas:
-    'thumbnail title'
-    'thumbnail timestamp';
-  grid-template-columns: auto 1fr;
-  grid-template-rows: min(auto, 2fr) 1fr;
+  grid-template:
+    'thumbnail title' 2fr
+    'thumbnail timestamp' 2fr / auto 1fr;
```

Comment:
Original value is `grid-template-rows: min(auto, 2fr) 1fr;`
Are the values right after change?

Reply:
Yes

Reply:
Ya the current value seems invalid
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/035fbac0-c5cc-46df-897b-53a82e7e361b)